### PR TITLE
temporarily disabling flaky test

### PR DESCRIPTION
**Pull request by CuriousVini**
Disabling flaky test temporarily so that build is not broken. 

Build: http://builds.cask.co/browse/CDAP-DUT4346-3

### CODE REVIEW

**Comment by sreevatsanraman** [comment]
Better way to do this is to add @Ignore annotation to the test. That way everyone would know the test is disabled. http://junit.sourceforge.net/javadoc/org/junit/Ignore.html

Also, please add a TODO comment next to the annotation with a blocker Jira for 3.5 to fix the test.

File: cdap-app-templates/cdap-etl/cdap-data-pipeline/src/test/java/co/cask/cdap/datapipeline/DataPipelineTest.java [hunk 1]
```diff
@@ -205,10 +205,10 @@ public void testInnerJoin() throws Exception {
     DataSetManager<Table> sinkManager = getDataset("joinerOutput");
     Set<StructuredRecord> expected = ImmutableSet.of(joinRecordSamuel, joinRecordJane);
     Set<StructuredRecord> actual = Sets.newHashSet(MockSink.readOutput(sinkManager));
-    Assert.assertEquals(expected, actual);
-
-    validateMetric(2, appId, "testJoiner.records.out");
-    validateMetric(2, appId, "sink1.records.in");
```
Comment (rohitsinha54) [inline cdap-app-templates/cdap-etl/cdap-data-pipeline/src/test/java/co/cask/cdap/datapipeline/DataPipelineTest.java]: Put a TODO and jira id (i guess there is an issue for this flaky test) here to enable it.

**Comment by CuriousVini** [comment]
@sreevatsanraman Addressed comments. Please take another look. Thanks!

**Comment by sreevatsanraman** [comment]
Lgtm

**Comment by CuriousVini** [comment]
The build is green. Merging.

**Pull request merged**